app/src/main/java/ai/elimu/appstore/model/ApplicationVersion.java

```diff
@@ -31,6 +31,9 @@
     @NotNull
     private String fileUrl;
 
+    @NotNull
```

Comment:
`@NotNull` will cause previous versions to crash due to missing data in this column:

```
java.lang.RuntimeException: Unable to create application ai.elimu.appstore.BaseApplication: android.database.sqlite.SQLiteConstraintException: NOT NULL constraint failed: APPLICATION_VERSION.CHECKSUM_MD5 (code 1299)
```

This is fine for now, but as of next month we need to handle database migrations properly for every release.